src/transaction/types/input.rs

```diff
@@ -175,7 +209,15 @@ impl Input {
     }
 
     pub const fn is_coin(&self) -> bool {
-        matches!(self, Input::Coin { .. })
+        matches!(self, Input::CoinSigned { .. } | Input::CoinPredicate { .. })
```

Comment:
would it be cleaner to have
```
self.is_coin_signed() || self.is_coin_predicate()
```
?

Reply:
Done! Lost the track of the git commit since it was rebased